src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModellerGermlineSparkToggleIntegrationTest.java

```diff
@@ -0,0 +1,398 @@
+                                                                                                                             final String sampleSexGenotype) {
+        final List<Target> targets = truth.targets();
+        int targetIndex = 0;
+
+        final int[] truthCopyNumberCallsPerSample = Arrays.stream(truth.getColumn(sampleIndex))
+                .mapToInt(d -> (int)d).toArray();
+        final int[] inferredCopyNumberCallsPerSample = Arrays.stream(inferred.getColumn(sampleIndex))
+                .mapToInt(d -> (int)d).toArray();
+
+        final int[] correctCopyNumberCallCounts = new int[MAX_COPY_NUMBER + 1];
+        final int[] truthCopyNumberCallCounts = new int[MAX_COPY_NUMBER + 1];
+        int truthRefCallCounts = 0, truthAltCallCounts = 0;
+        int correctRefCallCounts = 0, correctAltCallCounts = 0;
+        for (final Target target : targets) {
+            final int refCopyNumber = GERMLINE_PLOIDY_ANNOTATIONS.getTargetGermlinePloidyByGenotype(target,
+                    sampleSexGenotype);
+            final int truthCopyNumber = truthCopyNumberCallsPerSample[targetIndex];
+            final int inferredCopyNumber = inferredCopyNumberCallsPerSample[targetIndex];
+
+            /* ref/alt-blind concordance */
+            truthCopyNumberCallCounts[truthCopyNumber]++;
+            if (inferredCopyNumber == truthCopyNumber) {
+                correctCopyNumberCallCounts[truthCopyNumber]++;
+            }
+
+            /* ref/alt-resolved concordance */
+            if (truthCopyNumber == refCopyNumber) { /* ref truth */
+                truthRefCallCounts++;
+                if (inferredCopyNumber == truthCopyNumber) {
+                    correctRefCallCounts++;
+                }
+            } else { /* alt truth */
+                truthAltCallCounts++;
+                if (inferredCopyNumber == truthCopyNumber) {
+                    correctAltCallCounts++;
+                }
+            }
+            targetIndex++;
+        }
+
+        final double[] absConcordance = new double[MAX_COPY_NUMBER + 1];
+        for (int cn = 0; cn <= MAX_COPY_NUMBER; cn++) {
+            if (truthCopyNumberCallCounts[cn] > 0) {
+                absConcordance[cn] = ((double) correctCopyNumberCallCounts[cn]) / truthCopyNumberCallCounts[cn];
+            } else {
+                absConcordance[cn] = 1.0; /* truth has to be 0 too */
+            }
+        }
+        final double refConcordance = truthRefCallCounts > 0 ?
+                ((double)correctRefCallCounts) / truthRefCallCounts : 1.0;
+        final double altConcordance = truthAltCallCounts > 0 ?
+                ((double)correctAltCallCounts) / truthAltCallCounts : 1.0;
+
+        return new SampleCopyNumberConcordanceSummary(refConcordance, altConcordance, absConcordance);
+    }
+
+    private ReadCountCollection loadTruthCopyNumberTable(@Nonnull final File truthCopyNumberFile,
```

Comment:
Test code can skip the `try. . . catch` by declaring that the method throws an exception.